Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -534,3 +584,20 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         return df
     }()
 }
+
+private extension UICollectionView {
+    var isLastCellVisible: Bool { isLastCellFullyVisible }
```

Comment:
We have multiple options when the cell is visible. Any part (even 1px), fully visible or something in between. I tested it and the fully looks good. It's shown on the video in the PR description.

Reply:
Other options listed for visibility, will remove them when there are no comments on this.

Reply:
Sounds good 👍 If you think the UX feels right let's use `isLastCellFullyVisible`